README.md

```diff
@@ -3,7 +3,7 @@
 
 Code repository for 2024 Data Science for the Common Good project with iNaturalist. 
 
-Collaborators: Angela Zhu, Paula Navarrete, Sergei Pogorelov, Ozel Yilmazel 
+Collaborators: Angela Zhu, Paula Navarrete, Sergei Pogorelov, Ozzie 
```

Comment:
my name is wrong

Reply:
I don’t know where it came from again, I updated it